.github/workflows/pull-requests.yml

```diff
@@ -590,23 +589,232 @@ jobs:
+          else
+            echo "Currency L0 - 2 not health or Ready:"
+            curl -s http://127.0.0.1:9500/node/info
+            exit 1
+          fi
           echo ""
           
-          echo "Currency L0"
-          curl http://127.0.0.1:9400/cluster/info
+          echo "Currency L0 - 3"
+          if curl -s http://127.0.0.1:9600/node/info | grep -o "\"state\":\"Ready\"" >> /dev/null
+          then
+            echo "Currency L0 - 3 health and Ready"
+          else
+            echo "Currency L0 - 3 not health or Ready:"
+            curl -s http://127.0.0.1:9600/node/info
+            exit 1
+          fi
+          echo ""  
+
+      - name: Check Currency L1 nodes health
+        run: |
+          echo "Currency L1 - 1"
+          if curl -s http://127.0.0.1:9700/node/info | grep -o "\"state\":\"Ready\"" >> /dev/null
+          then
+            echo "Currency L1 - 1 health and Ready"
+          else
+            echo "Currency L1 - 1 not health or Ready:"
+            curl -s http://127.0.0.1:9700/node/info
+            exit 1
+          fi
           echo ""
           
-          echo "Currency L1"
-          curl http://127.0.0.1:9700/cluster/info
+          echo "Currency L1 - 2"
+          if curl -s http://127.0.0.1:9800/node/info | grep -o "\"state\":\"Ready\"" >> /dev/null
+          then
+            echo "Currency L1 - 2 health and Ready"
+          else
+            echo "Currency L1 - 2 not health or Ready:"
+            curl -s http://127.0.0.1:9800/node/info
+            exit 1
+          fi
           echo ""
+          
+          echo "Currency L1 - 3"
+          if curl -s http://127.0.0.1:9900/node/info | grep -o "\"state\":\"Ready\"" >> /dev/null
+          then
+            echo "Currency L1 - 3 health and Ready"
+          else
+            echo "Currency L1 - 3 not health or Ready:"
+            curl -s http://127.0.0.1:9900/node/info
+            exit 1
+          fi
+          echo ""      
+
+      - name: Check Currency L0 nodes ordinals
+        run: |
+          export CL0_1_ORDINAL=$(curl -s http://127.0.0.1:9400/snapshots/latest -H "Accept: application/json" | grep -o -m 1 "\"ordinal\":[0-9]*")
+          export CL0_2_ORDINAL=$(curl -s http://127.0.0.1:9500/snapshots/latest -H "Accept: application/json" | grep -o -m 1 "\"ordinal\":[0-9]*")
+          export CL0_3_ORDINAL=$(curl -s http://127.0.0.1:9600/snapshots/latest -H "Accept: application/json" | grep -o -m 1 "\"ordinal\":[0-9]*")
+
+          if [[ $CL0_1_ORDINAL == $CL0_2_ORDINAL && $CL0_1_ORDINAL == $CL0_3_ORDINAL ]]; then
+            echo "Ordinals are the same:"
+            echo "CL0 - 1: $CL0_1_ORDINAL"
+            echo "CL0 - 2: $CL0_2_ORDINAL"
+            echo "CL0 - 3: $CL0_3_ORDINAL"
+          else
+            echo "Ordinals are not the same:"
+            echo "CL0 - 1: $CL0_1_ORDINAL"
+            echo "CL0 - 2: $CL0_2_ORDINAL"
+            echo "CL0 - 3: $CL0_3_ORDINAL"
+            exit 1
```

Comment:
ordinals may be not the same quite often, so this won't be a deterministic test.

Reply:
Hmm, how to you recommend testing this? Ryle as said to catch the latest one